src/ape/_cli.py

```diff
@@ -86,15 +88,13 @@ def get_command(self, ctx, name):
             try:
                 return self.commands[name]()
             except Exception as err:
-                notify(
-                    "WARNING",
-                    f"Unable to load CLI endpoint for plugin 'ape_{name}'.\n\t{err}",
-                )
+                logger.warning(f"Unable to load CLI endpoint for plugin 'ape_{name}'.\n\t{err}")
```

Comment:
Unable to use `ctx.obj.log_warning()` here because this happens before the context object exists